tests/unittest/disaggregated/test_router.py

```
@@ -854,6 +855,97 @@ async def test_kv_cache_aware_router_polls_kv_cache_events(
+    router = KvCacheAwareRouter(
+        server_role=None,
+        servers=servers,
+        use_tokens=False,
+        max_batch_size=64,
+        tokens_per_block=32,
+        backfill_block_hashes_on_finish=False,
+    )
+    router._kv_signal_warn_threshold = 4
+
+    with caplog.at_level(logging.WARNING):
+        for i in range(router._kv_signal_warn_threshold + 3):
```

Ruff (0.15.18)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)